CHANGELOG.md

```diff
@@ -64,6 +64,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - `%files from` in a definition file will now correctly copy symlinks that point
   to a target above the destination directory, but inside the destination stage
   rootfs.
+- Fix a regression issue that overwrites source image runscript, environment
```

Comment:
I will make some changes to this log entry when I backport it to the release-1.3 branch